gmusicapi/clients/musicmanager.py

```diff
@@ -385,7 +385,8 @@ def download_song(self, song_id):
 
     @utils.accept_singleton(basestring)
     @utils.empty_arg_shortcircuit(return_code='{}')
-    def upload(self, filepaths, transcode_quality='320k', enable_matching=False):
```

Comment:
don't have a strong opinion on it. I like this ordering better and we introduce lots of breaking changes anyway, so I'm fine with leaving it.